docs/kubernetes/cross-cluster.md

```diff
@@ -0,0 +1,229 @@
+
+    ```console
+    $ kubectl delete pvc test-pvc-fs
+
+    $ kubectl get pvc test-pvc-fs
+    Error from server (NotFound): persistentvolumeclaims "test-pvc-fs" not found
+
+    $ kubectl get pv pvc-88959393-01c7-415f-aac4-a974c45f7ec7
+    NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS     CLAIM                 STORAGECLASS    REASON   AGE
+    pvc-88959393-01c7-415f-aac4-a974c45f7ec7   1Ti        RWX            Retain           Released   default/test-pvc-fs   csi-filestore            29m
+    ```
+
+    Similarly, switch context to `ClusterB` and delete the PVC and check that the PV is not deleted.
+    ```console
+    $ kubectl get pv pre-pv
+    NAME     CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS     CLAIM                 STORAGECLASS    REASON   AGE
+    pre-pv   1Ti        RWX            Retain           Released   default/test-pvc-fs   csi-filestore            16m
+    ```
+
+    Delete the PersistentVolume from both the clusters.
+
+    From `ClusterB`
+    ```console
+    $ kubectl delete pv pre-pv
+    persistentvolume "pre-pv" deleted
+    ```
+
+    From `ClusterA`
+    ```console
+    $  kubectl delete pv pvc-88959393-01c7-415f-aac4-a974c45f7ec7
+    persistentvolume "pvc-88959393-01c7-415f-aac4-a974c45f7ec7" deleted
+    ```
+
+6. Since the PersistentVolumes were created with ReclaimPolicy `Retain`, when the PV is deleted, the underlying filestore instance will not be deleted. If the instance is not longer needed, it can be deleted using filestore delete API.
+
+    ```console
+     gcloud filestore instances list
+    INSTANCE_NAME                             ZONE           TIER      CAPACITY_GB  FILE_SHARE_NAME  IP_ADDRESS     STATE  CREATE_TIME
+    pvc-88959393-01c7-415f-aac4-a974c45f7ec7  us-central1-c  STANDARD  1024         vol1             10.159.102.90  READY  2020-12-15T04:30:18
+    ```
+
+    ```console
+    $ gcloud filestore instances delete pvc-88959393-01c7-415f-aac4-a974c45f7ec7 --zone us-central1-c
+    You are about to delete Cloud Filestore instance projects/mattcary-saikatroyc-test/locations/us-central1-c/instances/pvc-88959393-01c7-415f-aac4-a974c45f7ec7.
+    ```
+
+The above example can also be performed for a RecalimPolicy `Delete`. In that case, deletion of PVC will trigger deletion of PV and this would also trigger the filestore CSI driver to automatically delete the underlying filestore instance. So care must be taken, when using a `Delete` policy.
```

Comment:
Maybe make it more explicit and say "In that case, deletion of a PVC in just one of the clusters will trigger deletion of the PV and then the underlying filestore instance. The other cluster's PV will point to a nonexistent filestore instance, pods will start failing I/O and devopds will be sad. So, care must be taken when using a `Delete` policy.

Reply:
+1 - nice to have an explanation of what happens to the cluster that loses it's instance without knowing :)